validator/build.py

```diff
@@ -164,11 +164,11 @@ def GenValidatorProtoascii(out_dir):
   assert re.match(r'^[a-zA-Z_\-0-9]+$', out_dir), 'bad out_dir: %s' % out_dir
 
   protoascii_segments = [open('validator-main.protoascii').read()]
-  extensions = glob.glob('extensions/*/0.1/validator-*.protoascii')
+  extensions = glob.glob('extensions/*/(0.1|1.0)/validator-*.protoascii')
```

Comment:
That's a design decision that I haven't heard the validator team discuss. One plan we have discussed is to introduce versioning to the tagspecs themselves and also pull the protoascii out of the versioned file but leave the appropriate validator test files with their appropriate versions.